profile/README.md

```diff
@@ -1,3 +1,30 @@
 # California Cropland Monitoring and Modeling Framework
 
 ## About
+
+The California Cropland Monitoring and Modeling Framework (CCMMF) will be a biogeochemical model and data integration pipeline to generate inventories and future projections of soil carbon (SOC) stocks and greenhouse gas fluxes (GHG) from California’s croplands. 
+CCMMF will leverage Bayesian techniques to combine heterogeneous datasets into unified wall-to-wall inventories and climate and scenario-based projections. 
```

Comment:
"to combine model predictions and heterogenous datasets..." ?

Reply:
```suggestion
CCMMF will leverage Bayesian techniques to combine model predictions and heterogeneous datasets into unified wall-to-wall inventories and climate and scenario-based projections. 
```